docs/api/ast-utils.md

```diff
@@ -342,6 +342,8 @@ module.exports = {
         return {
             MemberExpression(node) {
                 const name = getPropertyName(node, context.getScope())
+                // When using ESLint>=8.37.0, write as follows:
+                // const name = getPropertyName(node, context.sourceCode.getScope(node))
```

Comment:
The preferred option should not be to use a deprecated api. so it could be:

When using ESLint<8.37.0, write as follows:
context.getScope().